docs/contributing/templates/c2c-source.md

```diff
@@ -0,0 +1,103 @@
+---
+id: c2c-source  #example: druva-source
+title: XYZ Source #example: Druva Source
+sidebar_label: XYZ #example: Druva
+description: Description goes here. #example: Learn how to configure the Druva Cloud-to-Cloud Source in your Sumo Logic environment.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+{{source icon}}
+
+The {{source name}} collects {{data/event types}} from {{source of origin}}. {{what the app does}}
```

Comment:
This can be added under ## Data types

Reply:
I noticed in looking through the C2C docs that this introductory paragraph and ## Data Types content appear redundant.